demos/tutorial/indices/find2_index_approx.cpp

```diff
@@ -0,0 +1,35 @@
+//![Complete]
+#include <seqan/index.h>
+
+using namespace seqan;
+
+int main()
+{
+    DnaString genome("GAGAGGCCACTCGCAGGATTAAGTCAATAAGTTAATGGCGTGGGGTTATGGTATGTAGTACGACGCCCACAGTGACCTCATCGGTGCATTTCCTCATCGTAGGCGGAACGGTAGACACAAGGCATGATGTCAAATCGCGACTCCAATCCCAAGGTCGCAAGCCTATATAGGAACCCGCTTATGCCCTCTAATCCCGGACAGACCCCAAATATGGCATAGCTGGTTGGGGGTACCTACTAGGCACAGCCGGAAGCA");
+    Index<DnaString, BidirectionalIndex<FMIndex<> > > index(genome);
+
+    //![Delegate]
+    auto delegate = [](auto & iter)
+    {
+        for (auto occ : getOccurrences(iter))
+            std::cout << occ << std::endl;
+    };
+    //![Delegate]
+
+    DnaString pattern("GGGGTTAT");
+    std::cout << "Hits with up to 2 errors:" << std::endl;
+    //![SinglePattern]
+    find<0, 2>(delegate, index, pattern, HammingDistance());
```

Comment:
the interface differs only from the regular one by the template parameters?

Reply:
Yes. I'd prefer if we wouldn't call it `find()`, maybe `findFast()`? 😜 The current find-module is already confusing...